### ISSUE TITLE
Sequential composition with OF13

### ISSUE THREAD

**Issue by ralvarep** [fp7-netide/Engine#111]
Hi @schwabe, I have started to test sequential composition using NAT + SimpleSwitch with OF13. My composition file is the following:

```
<?xml version="1.0" ?>
<CompositionSpecification
        xmlns="http://netide.eu/schemas/compositionspecification/v1">
    <Modules>
        <Module id="NAT" loaderIdentification="nat_13.py" noFenceSupport="true">
            <CallCondition events="packetIn" datapaths="1"/>
        </Module>
        <Module id="SimpleSwitch" loaderIdentification="simple_switch_13.py" noFenceSupport="true">
            <CallCondition events="packetIn" datapaths="1"/>
        </Module>
    </Modules>
    <Composition>
        <ModuleCall module="NAT"/>
        <ModuleCall module="SimpleSwitch"/>
    </Composition>
</CompositionSpecification>
```

When there is some network traffic in the topology the Java Core shows in the log this warning:
![captura de pantalla de 2016-07-28 09 44 12](https://cloud.githubusercontent.com/assets/3491919/17204815/de6c3d8e-54a7-11e6-8e0d-854c42fb3d06.png)
Do you think that there can be something wrong related to that warning?
Thanks!

**Comment by schwabe** [comment]
Serial composition is not implemented at all for 1.3. Decision on last face to face meeting was to concentrate on parallel composition for the implementation.

**Comment by ralvarep** [comment]
Ok @schwabe , thanks for the information! :+1:
